help/user-guide/copasi-button.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Using Copasi in FAIRDOM-SEEK
```

Comment:
So not using a custom layout, but taking out the mention of FAIRDOM SEEK here, since the site.title is adding FAIRDOM SEEK Documentation in from of the page title anyway for SEO. 


SOrry if this was all clear already!